recipe/meta.yaml

```diff
@@ -81,17 +79,18 @@ test:
     - skbio.tree.tests
     - skbio.util
     - skbio.util.tests
-  requires:
-    - nose
-    - httpretty
   commands:
     - python -m skbio.test  # [not linux]
     - DISPLAY=localhost:1.0 xvfb-run -a bash -c 'python -m skbio.test'  # [linux]
```

Comment:
@RNAer, I think the error comes from here. I think the better solution is to set `MPLBACKEND` instead of xvfb.